src/Http/Controllers/MailablesController.php

```diff
@@ -134,7 +134,7 @@ public function previewMailable($name)
 
     public function delete(Request $request)
     {
-        $mailableFile = config('maileclipse.mailables_dir').$request->mailablename.'.php';
+        $mailableFile = config('maileclipse.mailables_dir').'/'.$request->mailablename.'.php';
```

Comment:
Maybe add this as a different PR, as it is I am not sure if this relates to the feature you are adding.

Having a PR focused to a single topic is easier to track and manage.

Reply:
@ReeceM Thank you for this. I don't think I can remove this commit now. If you have admin rights could you please remove it so I can do a separate PR?

Reply:
Hi @UdyW, what you can do is just undo that edit to the repo and make a new commit on your fork, GitHub will sync that commit to this PR

Reply:
Done. Thanks

Reply:
Cool, thank. If you want you can make a separate branch on your local with `git checkout -b fix/mailable-delete`


Then you can push that and github will offer to create a new PR on the web interface

Reply:
Already did. Any idea when these will get released?

Reply:
Alright, must have been commenting when you did the PR 😄 

I am not sure when they would be released. 

Maybe @Qoraiche would be able to give you an idea of that. 👍